fe1-web/src/features/evoting/components/ElectionOpened.tsx

```diff
@@ -0,0 +1,102 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import { Text } from 'react-native';
+import { Badge } from 'react-native-elements';
+import { useToast } from 'react-native-toast-notifications';
+
+import { CheckboxList, TimeDisplay, WideButtonView } from 'core/components';
+import { FOUR_SECONDS } from 'resources/const';
+import STRINGS from 'resources/strings';
+
+import { castVote, terminateElection } from '../network/ElectionMessageApi';
+import { Election, SelectedBallots } from '../objects';
+
+const ElectionOpened = ({ election, questions, isOrganizer, canCastVote }: IPropTypes) => {
```

Comment:
`canCastVote` seems to be only a concern to this specific component.
Couldn't we compute it internally to this component ? 
Ideally, it'd be great to be able to do `election.canCastVote()`, but I understand this still requires a refactoring and is probably beyond the scope of this PR.

Reply:
Haha yes, I noticed that too and changed it in https://github.com/dedis/popstellar/pull/981! :) I will quickly port that change to this PR 👍

Reply:
`election.canCastVote()` would be cool indeed! :) I can to this in the same PR or in a separate one, whatever you prefer :) Maybe it is easiest if I do it in the PR for encrypting the votes?